lib/workers/repository/extract/index.spec.ts

```diff
@@ -29,6 +29,13 @@ describe('workers/repository/extract/index', () => {
       // FIXME: explicit assert condition
       expect(res).toMatchSnapshot();
     });
+
+    it('warns if no packages found for a enabled manager', async () => {
+      config.enabledManagers = ['npm'];
+      managerFiles.getManagerPackageFiles.mockResolvedValue([]);
+      await expect(extractAllDependencies(config)).resolves.not.toThrow();
```

Comment:
```suggestion
      expect(await extractAllDependencies(config)).toEqual([]);
```

You call also add an expect for `logger.warn` to have been called.